gcloud/streaming/test_transfer.py

```diff
@@ -1941,17 +1922,18 @@ def __call__(self, request, end):
         return self._response
 
 
-def _tempdir():
+def _tempdir_maker():
```

Comment:
I like this change (with the conflicting names). Did `pylint` complain or did you just scream out for sanity?